index.html

```diff
@@ -0,0 +1,397 @@
+<!DOCTYPE html>
+<!--[if lt IE 9]><html class="lte-ie8" lang="en"><![endif]-->
+<!--[if gt IE 8]><!-->
+<html class=" show-global-bar" lang="en">
+<!--<![endif]-->
+
+<head>
+  <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+  <meta charset="utf-8">
+  <title>Service unavailable - GOV.UK</title>
+
+  <!--[if gt IE 8]><!-->
+  <link rel="stylesheet" media="screen"
+    href="assets/govuk-template.css">
+  <!--<![endif]-->
+  <!--[if IE 8]><link rel="stylesheet" media="screen" href="https://www.gov.uk/assets/static/govuk-template-ie8-e4fff1a2ce0e93e66e96db6eedae992a3859b1e608aa46bd3e42ac30f61c58ac.css" /><![endif]-->
+  <link rel="stylesheet" media="print"
+    href="assets/govuk-template-print.css">
```

Comment:
Browser console's throwing an error at this line – file can't be found. 
I don't know why anyone would ever want to print an error page, but if someone happens to want to, we should probably include that file.

Reply:
This should be sorted now, you happy to resolve?